python/lsst/ts/cRIOpy/GUI/EventWindow.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of M1M3 SS GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from PySide2.QtCore import Slot
+from PySide2.QtWidgets import QWidget, QVBoxLayout, QTreeView
+from PySide2.QtGui import QStandardItemModel, QStandardItem
+
+
+class EventWindow(QWidget):
+    def __init__(self, comm, topic):
+        super().__init__()
+
+        self._data = None
+
+        layout = QVBoxLayout()
+        self.tree = QTreeView()
+        self.model = QStandardItemModel()
+        self.model.setHorizontalHeaderLabels(["Name", "Data"])
+        self.tree.setModel(self.model)
+
+        layout.addWidget(self.tree)
+
+        self.setLayout(layout)
+
+        getattr(comm, topic).connect(self.newData)
+
+        data = getattr(comm.remote, "evt_" + topic).get()
+        if data:
+            self.newData(data)
+
+    @Slot(map)
+    def newData(self, data):
+        if self._data is None:
+            for a in data.__dict__:
```

Comment:
Please consider to has a more meaningful name than `a`.

Reply:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/5473542/139741860-111e81ca-3ab1-42f5-97f0-c42198457ee1.png">